Remove the pylint mccabe extension until the Python mccabe module is
available in Userland/Solaris.

These changes will not be sent upstream (and are only temporary).

--- pylint-1.6.4/pylint/__pkginfo__.py.orig	2016-09-29 09:22:05.276892599 +0000
+++ pylint-1.6.4/pylint/__pkginfo__.py	2016-09-29 09:22:15.294658505 +0000
@@ -19,7 +19,6 @@
     'astroid>=1.4.5,<1.5.0',
     'six',
     'isort >= 4.2.5',
-    'mccabe',
 ]
 
 extras_require = {}
--- pylint-1.6.4/pylint.egg-info/requires.txt.orig	2016-09-29 09:23:04.147211978 +0000
+++ pylint-1.6.4/pylint.egg-info/requires.txt	2016-09-29 09:23:17.057516302 +0000
@@ -1,7 +1,6 @@
 astroid>=1.4.5,<1.5.0
 six
 isort >= 4.2.5
-mccabe
 
 [:python_version=="2.7"]
 configparser
